src/test/java/com/puppycrawl/tools/checkstyle/checks/indentation/IndentationCheckTest.java

```diff
@@ -160,6 +163,9 @@ private void verify(Configuration config, String filePath, String[] expected,
             final IndentComment... linesWithWarn) throws Exception {
         final Checker checker = createChecker(config);
         checker.addListener(new IndentAudit(linesWithWarn));
+        checker.setLocaleCountry(Locale.ROOT.getCountry());
+        checker.setLocaleLanguage(Locale.ROOT.getLanguage());
+        checker.finishLocalSetup();
```

Comment:
The `IndentationCheckTest` is quite different from other tests. There are many `verifyWarns` in test methods. The actual assertion is at https://github.com/checkstyle/checkstyle/blob/master/src/test/java/com/puppycrawl/tools/checkstyle/checks/indentation/IndentationCheckTest.java#L1785-L1786, that to use an `endsWith` but not `equals` to judge the correctness.

The `getExpectedMessage()` is defined here, https://github.com/checkstyle/checkstyle/blob/master/src/test/java/com/puppycrawl/tools/checkstyle/checks/indentation/IndentationCheckTest.java#L1818-L1827
And the error message in English could be one of the followings:
```
indentation.error.multi=''{0}'' have incorrect indentation level {1}, expected level should be one of the following: {2}.
indentation.child.error.multi=''{0}'' child have incorrect indentation level {1}, expected level should be one of the following: {2}.
indentation.error=''{0}'' have incorrect indentation level {1}, expected level should be {2}.
indentation.child.error=''{0}'' child have incorrect indentation level {1}, expected level should be {2}.
```
The `endWith` way could work when the message is in English or Chinese. But in many other languages, it will not work at all. The messages could have different postfixes due to the difference of singular-plural or even pronounce stress.
i.e. in Finnish
```
indentation.error={0} sisennyssyvyydellä {1} ei ole oikealla syvyydellä {2}.
indentation.error.multi=''{0}'' on virheellinen sisennystason {1}, odottaa tasolla pitäisi olla yksi seuraavista: {2}.
indentation.child.error.multi=''{0}'' lapsi on väärä sisennystason {1}, odottaa tasolla pitäisi olla yksi seuraavista: {2}.
indentation.child.error=''{0}'' lapsi on väärä sisennystason {1}, odottaa taso olisi {2}.
```
We can see the messages have totally different postfixes.

I have logged what information the `IndentComment` really has and found that it doesn't have the token type info at all.  It only has the actual and expected intent.
So we can only know that the message is smth like `''{an unknown token type}'' have incorrect indentation level {actual intent}, expected level should be {expected intent}.` I guess it is why the original contributor didn't use `equals` but `endWith`.

To sum up, I haven't found another way to do the assertion without `endWith`. And if we are using `endWith`, it is not able to do the assertion in many non-English languages. Therefore, I use the old/ugly way to enforce the locale in English.